src/main.ts

```diff
@@ -10,11 +10,11 @@ import { Bonjour, Service } from 'bonjour-service'
 import * as kittycad from '@kittycad/lib/import'
 
 // If it's not set, scream.
-const NODE_ENV = process.env.NODE_ENV
-if (!NODE_ENV) {
-  console.error('*FOX SCREAM* process.env.NODE_ENV is not explicitly set!')
-  process.exit(1)
-}
+const NODE_ENV = process.env.NODE_ENV || 'production'
+if (!process.env.NODE_ENV)
+  console.warn(
+    '*FOX SCREAM* process.env.NODE_ENV is not explicitly set!, defaulting to production'
+  )
```

Comment:
Ah yes we need this because the built package won't work I actually told jess about it a few days ago, nice catch!

I'm a little on the fence exposing this error to users but at the same time I can imagine anyone missing this argument and wondering what's going on. So I think it's good as-is.